### DISCUSSION TITLE
enhancer similar to magnific.ai

### DISCUSSION THREAD

**Discussion by kinower** [lllyasviel/Fooocus#3623] [General]
Is there a method of creating enhancer images similar to how magnific.ai does it?

**Comment by FCB-Sunny** [comment]
Yes, of course and we can use the stable diffusion image upscaler models.
And I'm the crazy fan of your webui forge and I have some questions.
I successed to use the flux in your webui forge but I have some errors to use the several extensions for FLUX.
For example, the regional prompter extension.
Did you have experience to use this extension for Flux lora?

**Reply by poor7** [reply]
Unfortunately it's not Forge, it's Fooocus.

**Reply by FCB-Sunny** [reply]
No, in webui forge, we can test flux model exactly. but my question is I can use the regional prompter with flux loras in webui forge

**Reply by poor7** [reply]
This way for you https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions
